setupSomeArtifacts.js

```diff
@@ -53,3 +54,172 @@ b
   .require(path.join(__dirname, 'static', 'dummy.js'), {expose: 'fs'})
   .bundle()
   .pipe(fs.createWriteStream(path.join(__dirname, 'static', 'stdlibBundle.js')));
+
+// === third task
+
+// Bundle BuckleScript compiler with ReasonReact .cmi and .cmj files:
+// 1. compile ReasonReact with bsb
+// 2. serialize ReasonReact .cmi and .cmj files
+// 3. bundle them with BuckleScript compiler (bs.js)
+
+/** Serialize a .cmi or a .cmj to a string that can be passed to
+  * ocaml.load_module */
+function serializeBinary(binFileName, jsFileName) {
```

Comment:
Damn, this is complex. Where does it come from? If this is mandatory, can you put an elaborate comment?

Reply:
Sorry, I forgot to document it. It's based on https://github.com/ocsigen/js_of_ocaml/blob/master/compiler/lib/js_output.ml#L279. I translated it to Reason, compiled with BuckleScript and then manually removed the bs-platform dependencies. I've updated the comment.